packages/bpk-tokens/src/transformDarkValues.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this buffer except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import through from 'through2';
+
+const calculatePotentialMerges = properties => {
+  const merges = {};
+
+  if (!properties) {
+    return merges;
+  }
+
+  properties.forEach(p => {
```

Comment:
Wouldn't a category specific for dyanmic colours be much nice than using the name as we have here?

Reply:
👍